lua/FTerm/terminal.lua

```diff
@@ -131,6 +132,7 @@ function Terminal:term()
 
         -- IDK what to do with this now, maybe later we can use it
         self.terminal = pid
+        self.jobid = vim.b.terminal_job_id
```

Comment:
Is `vim.b.terminal_job_id` any different than `pid` returned by `termopen()`? Last time I was also able to use `nvim_chan_send` with `pid`.

Reply:
If you open a terminal a do `echo b:terminal_job_id` and `echo b:terminal_job_pid` you will see that they are not the same. The former is Neovim's internal job id number. It is typically low as it starts count at 1 with nvim startup. The latter is the system process id, which corresponds to *nix pid. It is typically a bigger number.

Note though, I can imagine things are a bit different on windows where I don't know if a pid system exist(?). Anyway, `nvim_chan_send` only responds to `job_id` on my machine. I was not able to get the program working using the `self.terminal` which I guess corresponds to system process id.